CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 - Kill unnecessary ocamlformat processes with sigterm rather than sigint or
   sigkill (#1343)
+- Update CI Badge link (#1347)
```

Comment:
Not sure that it needs a Entry Changes (you can add the label `no changelog`)

Reply:
Thanks for the catch. I'll undo the commit for the changelog